serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -71,6 +71,14 @@ xref:security/authention-support-for-openapi-services.adoc[Authentication for Op
 Authentication for OpenAPI services in Serverless Workflow
 --
 
+[.card]
+--
+[.card-title]
+xref:security/orchestrating-third-party-services-with-oauth2.adoc[Orchestrating third party services with OAuth2 authentication]
+[.card-description]
+Orchestrating third party services with OAuth2 authentication
```

Comment:
```suggestion
Orchestration of third-party services using OAuth 2.0 authentication
```

Reply:
@hmanwani-rh yes, looks good. 
I didn't add your suggested change directly because the change must be applied in other files too, but your suggestion was added on my additional commit.
Thanks.